easybuild/easyblocks/i/intel_compilers.py

```diff
@@ -129,3 +130,17 @@ def make_module_req_guess(self):
             'LIBRARY_PATH': libdirs,
         }
         return guesses
+
+    def make_module_extra(self):
+        """Additional custom variables for intel-compiler"""
+        txt = super(EB_intel_minus_compilers, self).make_module_extra()
+
+        # on Debian/Ubuntu, /usr/include/x86_64-linux-gnu needs to be included in $CPATH for Intel C compiler
+        out, ec = run_cmd("gcc -print-multiarch", simple=False)
+        if ec == 0 and out:
+            multiarch_inc_dir = os.path.join('/usr', 'include', out.strip())
```

Comment:
not sure if explicit /usr/include is the best solution here. You can also get this path from:
`gcc -E -Wp,-v -xc /dev/null 2>&1 | grep $(gcc -print-multiarch)$`
it's more complex but avoids the explicit naming.